### PR TITLE
Add og meta tags to starter pack detail

### PR DESCRIPTION
**Pull request by devinivy**
This adds opengraph meta tags to the detail page for starter packs.  As a followup we'll probably add the starter pack's name into the og:title.

### CODE REVIEW

File: bskyweb/templates/starterpack.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block html_head_extra -%}
+  <meta name="apple-itunes-app" content="app-id=xyz.blueskyweb.app, app-clip-bundle-id=xyz.blueskyweb.app.AppClip, app-clip-display=card">
```
Comment (haileyok) [inline bskyweb/templates/starterpack.html:4]: 🙏

File: bskyweb/cmd/bskyweb/server.go [hunk 2]
```diff
@@ -377,6 +395,30 @@ func (srv *Server) WebPost(c echo.Context) error {
...
+			return c.Render(http.StatusOK, "starterpack.html", data)
+		}
+		handleOrDIDParam := c.Param("handleOrDID")
+		handleOrDID, err := syntax.ParseAtIdentifier(handleOrDIDParam)
+		if err != nil {
+			return c.Render(http.StatusOK, "starterpack.html", data)
+		}
+		identifier := handleOrDID.Normalize().String()
+		data["imgThumbUrl"] = fmt.Sprintf("%s/start/%s/%s", srv.cfg.ogcardHost, identifier, rkey)
+		log.Infof("hi %s", data["imgThumbUrl"])
```
Comment (devinivy) [inline bskyweb/cmd/bskyweb/server.go:417]:
```suggestion
```

File: bskyweb/templates/starterpack.html (hunk 1 above)
Comment (haileyok) [reply bskyweb/templates/starterpack.html:4]: just checking, is this only being added on the `/start/*` route?

Comment (devinivy) [reply bskyweb/templates/starterpack.html:4]: Yeah that's right, this template is only wired-up to the new `server.WebStarterPack` handler for that route.  The routing looks like this:
```go
e.GET("/start/:handleOrDID/:rkey", server.WebStarterPack)
```

File: .github/workflows/build-and-push-bskyweb-aws.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - divy/bskylink
+      - divy/starterpack-meta
```
Comment (haileyok) [inline .github/workflows/build-and-push-bskyweb-aws.yaml:7]: not sure if this is ready to go but just pointing to this if it is.

**Review by haileyok — APPROVED** [review]

File: bskyweb/cmd/bskyweb/server.go [hunk 4]
```diff
@@ -223,9 +240,9 @@ func serve(cctx *cli.Context) error {
 	e.GET("/profile/:handleOrDID/post/:rkey/liked-by", server.WebGeneric)
 	e.GET("/profile/:handleOrDID/post/:rkey/reposted-by", server.WebGeneric)
 
-    // starter packs
```
Comment (haileyok) [inline bskyweb/cmd/bskyweb/server.go:226]: mfw i have to remember to use a tab 😅 thanks

Comment (devinivy) [reply bskyweb/cmd/bskyweb/server.go:226]: When in doubt you can always use `go fmt`!

File: .github/workflows/build-and-push-bskyweb-aws.yaml (hunk 3 above)
Comment (devinivy) [reply .github/workflows/build-and-push-bskyweb-aws.yaml:7]:
```suggestion
```

**Pull request merged**